spec/mixlib/cli_spec.rb

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 #
 
-require File.expand_path(File.join(File.dirname(__FILE__), "..", "spec_helper"))
+require File.expand_path(File.join(__dir__, "..", "spec_helper"))
```

Comment:
```suggestion
require "spec_helper"
```

rspec puts `spec/` in the $LOAD_PATH so this should work.